src/azul/__init__.py

```diff
@@ -552,32 +552,36 @@ def indexer_concurrency(self):
         return int(os.environ['AZUL_INDEXER_CONCURRENCY'])
 
     @property
-    def notify_queue_name(self):
-        return self.qualified_resource_name('notify')
+    def notifications_queue_name(self):
+        return self.qualified_resource_name('notifications')
 
     @property
-    def document_queue_name(self):
-        return config.qualified_resource_name('documents', suffix='.fifo')
+    def tallies_queue_name(self):
```

Comment:
I thought about several options for reducing the combinatorial explosion in the number of properties. Please try this approach

```suggestion
    def tallies_queue_name(self, retry=False, fail=False):
        assert not (retry and fail)
```

I think that would be a net win, considering that it replaces five properties with two methods and makes the comprehension in the TF config easier.

Reply:
I did this and I agree it's improvement. I think more consolidation is possible by parametrizing all queue names into a single function, but I'll wait for your judgement before trying that out.